packages/eslint-plugin-js/package.json

```
@@ -101,7 +101,7 @@
     "rules/**/*.d.ts"
   ],
   "engines": {
-    "node": "^16.0.0 || >=18.0.0"
+    "node": "^18.8.0 || >=20.0.0"
```

```suggestion
    "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
```

Most of the other ESLint-Community projects seems to align with ESLint 9 and use this range instead

If you want to ensure it's not too wide you can do so automatically with my: https://www.npmjs.com/package/installed-check